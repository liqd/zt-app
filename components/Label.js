import React from 'react';
import { View, Text } from 'react-native';
import { styles } from './Label.styles';

export const Label = (props) => {
  return (
    <View style={styles.label}>
      <Text style={styles.text}>{props.title}</Text>
    </View>
  );
};
