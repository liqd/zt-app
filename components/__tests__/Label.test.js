import React from 'react'
import { render } from '@testing-library/react-native'
import { Label } from '../Label'

test('Test Label', () => {
  const { toJSON } = render(<Label title="TestTitle" />)
  expect(toJSON()).toMatchSnapshot()
})
