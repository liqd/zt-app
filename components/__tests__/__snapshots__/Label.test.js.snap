// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Label 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "alignSelf": "flex-start",
      "backgroundColor": "#1a1a1a",
      "borderRadius": 6,
      "marginBottom": 10,
      "marginRight": 10,
      "paddingHorizontal": 12,
      "paddingVertical": 4,
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "fontFamily": "SourceSansPro_400Regular",
        },
        Object {
          "color": "#fff",
          "fontSize": 15.9375,
        },
      ]
    }
  >
    TestTitle
  </Text>
</View>
`;
